code/modules/projectiles/guns/flamer.dm

```diff
@@ -585,46 +559,31 @@ GLOBAL_LIST_EMPTY(flamer_particles)
 
 	firelevel -= 2 //reduce the intensity by 2 per tick
 
-
 // override this proc to give different idling-on-fire effects
 /mob/living/flamer_fire_act(burnlevel)
 	if(!burnlevel)
 		return
-	var/fire_mod = get_fire_resist()
-	if(fire_mod <= 0)
-		return
 	if(status_flags & (INCORPOREAL|GODMODE)) //Ignore incorporeal/invul targets
 		return
-	if(hard_armor.getRating("fire") >= 100)
-		to_chat(src, span_warning("Your suit protects you from most of the flames."))
-		adjustFireLoss(rand(0, burnlevel * 0.25)) //Does small burn damage to a person wearing one of the suits.
+	if(hard_armor.getRating(FIRE) >= 100)
+		to_chat(src, span_warning("Your suit protects you from the flames."))
```

Comment:
This is a moderate buff to surt. If it's intended, you should mention it in the changelog.

Reply:
Minor buff to surt, aye. I'll get around to reworking fire code entirely at some point